docs/recipes/hitl.md

```diff
@@ -2,45 +2,49 @@
 
-[@botpress/hitl](https://github.com/botpress/botpress/tree/master/packages/functionals/botpress-hitl) is designed to solve this issue (only supports messenger-channel for now).
+The [Human-in-the-Loop (@botpress/hitl)](https://github.com/botpress/botpress/tree/master/packages/functionals/botpress-hitl) module allows you to do just that! (It does only support messenger-channel for now).
 
-Once you have this package installed (`npm install --save @botpress/hitl`) you'll be able to:
+Once you have this module installed (`npm install --save @botpress/hitl`) you will be able to:
 
-1. Pause conversation with bot
-2. Allert agents that conversation requires attention
-3. As an agent you'll be able to communicate with users through admin-panel
-4. Resume conversation programmatically
+1. Pause a users conversation with the bot
+2. Alert your agents that a conversation requires attention
+3. As an agent you will be able to continue the conversation via the admin-panel
+4. Resume conversation with the bot
 
 # Pausing conversation
 
-There are several ways you can pause conversation:
-- from admin UI toggling appropriate button
-- by performing API-requests:
+There are several ways you can pause the conversation:
+- from the admin-panel, toggling the appropriate button
+- by performing an API-request:
   - POST /api/botpress-hitl/sessions/{$id}/pause
   - POST /api/botpress-hitl/sessions/{$id}/unpause
 - programmatically by calling `bp.hitl.pause(platform, userId)` and `bp.hitl.unpause(platform, userId)`
 
-# Allering agents
+# Alerting agents
 
-You may have different ways of notifying agents with emails or external API's and this is all possible. But we'll cover a simple case of notifying agent via notification in admin-panel:
+There are a number of ways to alert your agents of a paused conversation, an email, a call to an external API or, as in the example below, via a notification in the admin-panel:
 
 ```js
   const message = event.user.first_name + ' wants to talk to a human'
   bp.notifications.send({ message, level: 'info', url: '/modules/botpress-hitl' })
 ```
 
-Agents can then navigate to appropriate conversation and start communication.
+The agent can then navigate to the appropriate conversation and take over the conversation from the bot.
 
 # Resuming conversation
 
-Once they agents are done communicating with the user they can unpause conversation. But this can be implemented programmatically either to allow user unpause conversation himself like this: `bp.hitl.unpause(platform, userId)`
+Once the agent is done communicating with the user, they can unpause the conversation. 
+
+However it is possible for the user to unpause the conversation programmatically by triggering an an action that calls `bp.hitl.unpause(platform, userId)`
```

Comment:
> an an action

> However it is possible for the user to unpause the conversation programmatically

I'd say "it is possible to allow the user to unpause the conversation programmatically"
It isn't possible unless developers implement this.